.github/workflows/build-nightly.yml

```
@@ -28,6 +28,11 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Generate nightly version number
+        id: version_vars
+        run: |
+          echo mastodon_nightly_version=nightly-$(TZ="Etc/UTC" date +'%Y-%m-%d') >> $GITHUB_OUTPUT
```

Should this be quoted?
```suggestion
          echo "mastodon_nightly_version=nightly-$(TZ="Etc/UTC" date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
```

---

Might need to change in the inner `'` for TZ if it get's wrapped

---

We can quote it, but please ensure it works, last time I tried to add quotes I could no figure how to do it, between YAML's quotes, shell escaping…

---

Probably best to keep it as is then, I'd rather not mess about with yaml weirdness — it works as is